pages/[username]/posts/[postId].tsx

```diff
@@ -78,6 +78,7 @@ export const PostPage = ({ note, name, profilePicture }: Props) => {
               <h1 className="text-center text-5xl font-bold leading-tight">
                 {note.title}
               </h1>
+              <p>Last Published: {formatNumber(note.publishedAt)}</p>
```

Comment:
We won't be needing this here. We can show the date inside the publish modal but no need for it on the main screen.

Reply:
Got it!